packages/tailwind/README.md

```diff
@@ -0,0 +1,5 @@
+# Install
```

Comment:
Should include examples of tailwind config and VSCode recommendations

Reply:
And a huge disclaimer that API is subject to change lol

Reply:
True that, there's some guidance in the FE Template I should link back too as well